pcsx2/CDVD/CdRom.cpp

```diff
@@ -844,6 +841,26 @@ void cdrWrite1(u8 rt)
 			cdr.Mode = cdr.Param[0];
 			cdr.Ctrl |= 0x80;
 			cdr.Stat = NoIntr;
+			if (cdr.Mode && MODE_CDDA)
+			{
+				StopCdda();
+				cdvd.Type = CDVD_TYPE_CDDA;
+				if (cdr.Reading == 1)
```

Comment:
why an == 1? isn't a standard if sufficient in this case or do we want the 1? If we do, why not an enum?